priv/templates/top-aside/top-aside.acl_collaboration_group.tpl

```diff
@@ -1,9 +1,7 @@
 <div class="u-d-flex u-flex-col u-flex-gap-2">
-    {% with m.search.paged[{query is_published content_group=id sort=-"modified" cat=['contribution', 'event'] id_exclude=upcoming_events|make_list pagelen=1 page=q.page}] as result %}
-        {% if result[1] as latest_contribution %}
-            {% catinclude "keywords/status-tags.tpl" latest_contribution %}
-        {% endif %}
-    {% endwith %}
+    {% if m.search[{query cat=['contribution', 'event'] sort="-rsc.created" is_published content_group=id pagelen=15}]|make_list|kc_collaboration_group_progress_label as result %}
```

Comment:
One suggestion for the use of the filter: since this is specifically about calculating the progress label for knowledge groups, I'd give the group's ID as parameter only:
```suggestion
    {% if id|kc_collaboration_group_progress_label as result %}
```